recipe/meta.yaml

```diff
@@ -10,9 +10,9 @@ source:
   sha256: 94f643f1edfaccb57d64b503c7c96f00dec64e8635c054bbaa33855d72c5822d
 
 build:
-  number: 0
+  number: 1
   features:
-    - vc9  # [win and py27]
+    - vc10  # [win and py27]
```

Comment:
This doesn't really work like this unfortunately. Changing the feature name won't change the compiler used, which still will be VS 2008 because of the Python 2.7 part. Maybe you are meaning to drop this?

Reply:
uunnn, sorry, I did not know about that. Therefore I think we have to exclude this part

Reply:
No worries. It's a bit technical unfortunately. Sorry for the confusion.